web/source/osk/visualKeyboard.ts

```diff
@@ -235,12 +261,11 @@ namespace com.keyman.osk {
 
     // Produces a HTMLSpanElement with the key's actual text.
     protected generateKeyText(osk: VisualKeyboard): HTMLSpanElement {
-      let util = (<KeymanBase>window['keyman']).util;
       let spec = this.spec;
 
       // Add OSK key labels
       var keyText;
-      var t=util._CreateElement('span'), ts=t.style;
+      var t = document.createElement('span'), ts=t.style;
```

Comment:
For reference: https://github.com/keymanapp/keyman/blob/e581d495bb3b31caf09e96632209484441cd67ec/web/source/kmwutils.ts#L314-L328

Reply:
Also:

https://github.com/keymanapp/keyman/blob/e581d495bb3b31caf09e96632209484441cd67ec/web/source/resources/osk/kmwosk.css#L53

https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#syntax

> **none**
> The text of the element and its sub-elements is not selectable. Note that the Selection object can contain these elements.

Any potential lingering effects from `util._CreateElement` -> `document.createElement` within this PR can be managed via our OSK CSS.

Reply:
> Any potential lingering effects

Are there any?

Reply:
If there are, it'd be with element selectability (all the `user-select` stuff), noting lines 322 through 325 from the linked `kmwutils.ts` code.

Based on the `kmwosk.css` code, I'd say "no" (assuming Konqueror has adjusted to use the the base `user-select` attribute) given the MDN link's documentation.  I haven't been able to trigger anything unwanted with my tests, though that may be due to the browsers I've tested with.